terraform/create.go

```diff
@@ -96,8 +97,15 @@ func (t *Terraform) Create() error {
 
 			t.updateConfig(ip, sshc, output)
 
+			// Upload service file
```

Comment:
One question. Why don't we launch every update of the instances concurrently using go routines?

Reply:
Keeping things simple for now. :) We can certainly move to this when we start deploying clusters.

Reply:
:+1: